ranger/container/directory.py

```diff
@@ -350,7 +351,15 @@ def load_bit_by_bit(self):
                     filenames = filelist
                     self.load_content_mtime = mtimelevel(mypath, self.flat)
                 else:
-                    filelist = os.listdir(mypath)
+                    try:
+                        filelist = os.listdir(mypath)
+                    except (OSError, IOError) as ex:
+                        # PermissionError is undefined in python 2
+                        if ex.errno == errno.EACCES:
+                            self.accessible = False
+                        else:
+                            raise
+                        filelist = []
```

Comment:
Can we move this assignment into the if block? I feel it'd make it more clear of our intent. It's not being done regardless of the if result, it just gets ignored in the else branch due to the exception.